### PR TITLE
Firefox 58 now implements "Promise.prototype.finally"

### PR DESCRIPTION
**Pull request by afmenez**
The latest Firefox Nightly now passes all the "Promise.prototype.finally" tests.
https://bugzilla.mozilla.org/show_bug.cgi?id=1019116

### CODE REVIEW

File: data-esnext.js [hunk 1]
```diff
@@ -2676,7 +2676,8 @@ exports.tests = [
       res: {
         babel: babel.corejs,
         typescript: typescript.corejs,
-        firefox52: false,
+        firefox2: false,
```
Comment (ljharb) [inline data-esnext.js]: Did you explicitly test Firefox 2? If not, it shouldn’t be in here.

Either way, please keep the explicit v52 results.

**Review by ljharb — CHANGES_REQUESTED** [review]

File: data-esnext.js (hunk 1 above)
Comment (afmenez) [reply data-esnext.js]: Yes, I have tested firefox 2 and every ESR version up to 52, and also versions 53..58.  I have also tested with many versions of Chrome and Edge.  I don't see the point of adding every version.

Comment (ljharb) [reply data-esnext.js]: Generally, these entries should represent explicit human test results, and thus should never be removed (unless incorrect).

I agree having all of those versions in the list is impractical; maybe we should settle for including 2, 57, and 58?

Comment (afmenez) [reply data-esnext.js]: Done.

**Review by ljharb — APPROVED** [review]
Thanks!

**Review by chicoxyzzy — APPROVED** [review]

**Comment by afmenez** [comment]
This was Firefox 59 nightly actually, I will fix it with the 57 release patch.
